packages/amplication-client/src/CommandPalette/CommandPalette.tsx

```diff
@@ -101,21 +102,27 @@ type Props = {
 };
 
 const CommandPalette = ({ trigger }: Props) => {
-  const match = useRouteMatch<{ resourceId: string }>("/:resourceId/");
+  const { currentWorkspace, currentProject, currentResource } = useContext(
+    AppContext
+  );
 
-  const { resourceId } = match?.params || {};
+  const projectBaseUrl = useMemo(
```

Comment:
we need to add a generic function to create this url and check for undefined entities.

Reply:
I agree, I event think it can be part of the context